docs/snippets/angular/button-story-default-docs-code.ts.mdx

```diff
@@ -0,0 +1,37 @@
+```ts
+// Button.stories.ts
+
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import Button from './button.component';
+
+export default {
+  title: 'Button',
+  component: Button,
+  argTypes: {
+    backgroundColor: { control: 'color' }
+  }
+} as Meta;
+
+// some function to demonstrate the behavior
+const someFunction = (someValue: string) => {
+  return `i am a ${someValue}`;
+};
+
+export const ExampleStory: Story<ComplexCase> = (args: ComplexCase) => {
```

Comment:
Where does the `ComplexCase` type come from for this?

Reply:
Thanks for catching this. `ComplexCase` was the name of the component that i was using . Fixed with the latest commit. Thank you for checking it :)